src/map.cpp

```diff
@@ -3845,8 +3845,8 @@ void map::smash_items( const tripoint_bub_ms &p, int power, const std::string &c
     if( items_destroyed > 1 ) {
         add_msg_if_player_sees( get_map().get_bub( get_abs( p ) ), m_bad,
                                 _( "The %s destroys several items!" ), cause_message );
-    } else if( items_destroyed == 1 && items_damaged == 1 ) {
         //~ %1$s: the cause of destruction, %2$s: destroyed item name
+    } else if( items_destroyed == 1 && items_damaged == 1 ) {
         add_msg_if_player_sees( get_map().get_bub( get_abs( p ) ), m_bad,
                                 _( "The %1$s destroys the %2$s!" ), cause_message,
                                 damaged_item_name );
```

Comment:
oh, that's actually a mistake of mine, the comment belongs to `The %1$s destroys the %2$s!` message, so it should be put under this message

Reply:
That's confusing... If the comment should be placed after rather than before the "add_msg_if_player_sees" message, then that ought to apply to the next comment as well?

Reply:
the comment `//~ %1$s: the cause of destruction, %2$s: destroyed item name` belong to the message `"The %1$s destroys the %2$s!"`, one that contain both `%1$s` and `%2$s`; message `The %s destroys several items!` contain only `%s`, single string

Reply:
I think we're miscommunicating. We agree which message the misbehaving comment belong to.

My confusion stems from what I perceive (possibly incorrectly) is a suggestion to place the comment after rather than before the associated message, because if that's the case, the next comment, at line 3857, should logically also be moved after its associated message.

If that's not what you're suggesting, then I get the interpretation that the original comment placement was correct, which the test obviously didn't agree with.

I may well bark up the wrong tree, though.

Reply:
Hmm, my assumption was is that comments supposed to go after the original line, but checking some another examples, it seems the comment should go before the line
https://github.com/CleverRaven/Cataclysm-DDA/blob/e43389302cc002ed591ccfebc35ecb39c8ba6d91/src/diary.cpp#L598-L605

Reply:
Hm, this sounds like the original placement should be correct based on this, but yet the tools disagrees for some unknown reason. Is there a way to find out what would both appease the test tool and the tools used for translation?